receiver/filelogreceiver/README.md

```diff
@@ -33,6 +33,8 @@ Tails and parses logs from files.
 | `include_file_path`                 | `false`                              | Whether to add the file path as the attribute `log.file.path`.                                                                                                                                                                                                  |
 | `include_file_name_resolved`        | `false`                              | Whether to add the file name after symlinks resolution as the attribute `log.file.name_resolved`.                                                                                                                                                               |
 | `include_file_path_resolved`        | `false`                              | Whether to add the file path after symlinks resolution as the attribute `log.file.path_resolved`.                                                                                                                                                               |
+| `include_file_owner_name`           | `false`                              | Whether to add the file owner name as the attribute `log.file.owner.name`. Only supported for linux or solaris.  |
+| `include_file_group_name`           | `false`                              | Whether to add the file group name as the attribute `log.file.group.name`. Only supported for linux or solaris.  |
```

Comment:
Sorry to have overlooked this earlier but I think we should rename this to reflect that the group is related to the owner, not the file. e.g. `log.file.owner.group.name`